src/components/TransactionsTab/TransactionsTab.styles.tsx

```diff
@@ -108,6 +114,30 @@ export const LegendLine = styled.span`
   background: transparent;
 `;
 
+export const StyledTooltip = styled(Tooltip)<{ $isTooltip: boolean }>`
```

Comment:
When Select is active don't show tooltip.

![Screenshot 2023-07-05 at 21 23 50](https://github.com/airswap/airswap-web/assets/86911296/8eeca9bd-30a5-4f01-8c5c-b4f2450af0ce)

Reply:
fixed